roxie/ccd/ccdfile.cpp

```diff
@@ -1900,17 +1900,15 @@ class CResolvedFile : implements IResolvedFileCreator, implements ISDSSubscripti
     IMPLEMENT_IINTERFACE;
 
     CResolvedFile(const char *_lfn, const char *_physicalName, IDistributedFile *_dFile, RoxieFileType _fileType, IRoxieDaliHelper* _daliHelper, bool isDynamic, bool cacheIt, bool writeAccess, bool _isSuperFile)
-    : daliHelper(_daliHelper), lfn(_lfn), physicalName(_physicalName), dFile(_dFile), fileType(_fileType), isSuper(_isSuperFile)
+    : lfn(_lfn), physicalName(_physicalName), dFile(_dFile), fileType(_fileType), isSuper(_isSuperFile), daliHelper(_daliHelper)
     {
         cached = NULL;
         fileSize = 0;
-        fileCheckSum = 0;
```

Comment:
there is still a member variable which is serialized and accessed.  This doesn't look right.

Reply:
Good catch - fixed